tests/org.eclipse.swt.tests/JUnit Tests/org/eclipse/swt/tests/junit/Test_org_eclipse_swt_browser_Browser.java

```diff
@@ -762,7 +762,7 @@ private String getValidUrl() {
 	testLogAppend("PLUGIN_PATH: " + pluginPath);
 	// When test is run via Ant, URL needs to be acquired differently. In that case the PLUGIN_PATH property is set and used.
 	if (pluginPath != null) {
-		return pluginPath + "/data/testWebsiteWithTitle.html";
+		return "file://" + pluginPath + "/data/testWebsiteWithTitle.html";
```

Comment:
This feels like a hack to me.  If this truly a Path, one should properly convert it to a URI, e.g., `Path.of().toUri().toString()` so that `\` is properly handled and even things like ` ` are properly encoded.

Reply:
Indeed, thank you! I improved that with 5423f29. Local execution works fine with that change.